app/controllers/admin/other_case_types_report_controller.rb

```diff
@@ -0,0 +1,13 @@
+class Admin::OtherCaseTypesReportController < AdminController
+  def index
+    @cases_with_other_type = TribunalCase.with_other_case_type.order(created_at: :desc)
+  end
+
+  private
+
+  # We are reusing the same credentials we created for the upload problems report, for simplicity.
```

Comment:
Yeah, we can do that. I was a bit uneasy about the fact the credentials env variables are (name-wise) coupled to the original controller, and wanted to make the duplication very obvious, but you are right, if we move this to the superclass the coupling is even more evident. I will change this.